src/cab/useCases/__tests__/sortCabList.test.ts

```diff
@@ -1,37 +1,69 @@
 import { describe, expect, it } from 'vitest';
-import { Cab, sortCabsList } from '../CabServicePage';
+import { sortCabsList } from '../service/CabListModelService';
+import { CabStatus } from '../domain/cab';
+
+const cab = {
+    id: '0',
+    model: 'Tesla',
+    segment: 'Luxury',
+    trunk: 'L',
+    capacity: 5,
+    rating: 5,
+    ratings: 243,
+    minutesAway: 20,
+    eta: 20,
+    impediment: undefined,
+    status: 'ARRIVED' as CabStatus,
+    isInTrafficJam: false,
+};
 
 describe('Pre-check', () => {
-  it('sort by eta', () => {
-    const cabs = [
+    it('sort by eta', () => {
+        const cabs = [
+            { ...cab, minutesAway: 10 },
+            { ...cab, minutesAway: 2 },
+            { ...cab, minutesAway: 1 },
+            { ...cab, minutesAway: 5 },
+            { ...cab, minutesAway: 4 },
+        ];
+        const result = sortCabsList(cabs).map((cab) => cab.minutesAway);
+        expect(result).toEqual([1, 2, 4, 5, 10]);
+    });
+
+    it('When ETAs are equal, sort by whether the cab is in a traffic jam', () => {
```

Comment:
In the test case descriptions, it's better to strive for the "what > when" format:

```suggestion
    it('returns cards sorted by traffic load when ETAs are equal', () => {
```

(ditto throughout)